src/pages/lib/graphqlapi/mutate-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,598 @@ export const meta = {
   title: `Create, update, and delete application data`,
   description: `Learn more about how to create and update data using GraphQL APIs in Amplify`
 };
+
+In this guide, you will learn how to create, update, and delete your data using Amplify Libraries' GraphQL client.
+
+Before you begin, you will need:
+
+<InlineFilter filters={["js"]}>
+
```

Comment:
Flagging here that this js link goes to the new page - if that connect page is not ready when we publish we may want to point this to the existing getting started in the meantime. However, that page is close so may be worth waiting and publish together.